scala-thoughts.md

```diff
@@ -0,0 +1,29 @@
+
+The feeling of using Scala is slightly odd. Most things really well, but only because it
+"was made that way." Of course, that's the whole point of having a scalable language.
+You can feel domain-specific in a much larger range of places. Even so, it still feels
+a bit unsettling. Does it ever happen that accidental implicit casts cause huge issues, for example?
```

Comment:
I imagine the "issues" they cause are because the programmer expected one thing but got another, and given that it is still statically typed and compile-time checked I don't think you get the same issues you would in something like Python.